ntbs-service/Startup.cs

```diff
@@ -47,9 +47,14 @@ public void ConfigureServices(IServiceCollection services)
                 options.UseSqlServer(auditDbConnectionString)
             );
 
-            if (Configuration.GetValue<bool>(Constants.AUDIT_ENABLED_CONFIG_VALUE)) {
+            if (Configuration.GetValue<bool>(Constants.AUDIT_ENABLED_CONFIG_VALUE))
+            {
                 services.AddEFAuditer(auditDbConnectionString);
             }
+            else
+            {
```

Comment:
I realised that it was actually just logging to a file before when the auditing flag was set to false which is the Audit.NET default behaviour

Reply:
Ah, that explains the rando files ;) Thanks! :D